packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch

```diff
@@ -1,18 +1,31 @@
 --- charts-original/values.yaml
 +++ charts/values.yaml
-@@ -10,9 +10,8 @@
+@@ -10,13 +10,11 @@
  controller:
    name: controller
    image:
--    repository: k8s.gcr.io/ingress-nginx/controller
--    tag: "v0.46.0"
--    digest: sha256:52f0058bed0a17ab0fb35628ba97e8d52b5d32299fbc03cc0f6c7b9ff036b61a
+-    registry: k8s.gcr.io
+-    image: ingress-nginx/controller
 +    repository: rancher/nginx-ingress-controller
-+    tag: "nginx-0.46.0-rancher1"
+     # for backwards compatibility consider setting the full image url via the repository value below
+     # use *either* current default registry/image or repository format or installing chart by providing the values.yaml will fail
+     # repository:
+-    tag: "v0.47.0"
+-    digest: sha256:a1e4efc107be0bb78f32eaec37bef17d7a0c81bec8066cdf2572508d21351d0b
++    tag: "nginx-0.47.0-hardened1"
      pullPolicy: IfNotPresent
      # www-data -> uid 101
      runAsUser: 101
-@@ -45,7 +44,7 @@
+@@ -26,7 +24,7 @@
+   existingPsp: ""
+ 
+   # Configures the controller container name
+-  containerName: controller
++  containerName: rke2-ingress-nginx-controller
```

Comment:
this is just the container within the pod; curious why this needed to be changed.

Reply:
it is, they changed to `- name: {{ .Values.controller.containerName }}` in ./packages/rke2-ingress-nginx/charts/templates/controller-{daemonset,deployment}.yaml

Reply:
I believe it is a leftover from upgrading older versions and the RKE2 CI checks for containers with specific names